src/branchandbound.jl

```diff
@@ -68,3 +68,25 @@ function divide_dom!(p::Union{TaylorN{T}, Taylor1{T}},
     R = append!(RR, R[(index + 1):length(R)])
     return D, R
 end
+
+function enclose_binary(f, dom; kmax=3, tol=1e-3)
+    y = f(dom)
```

Comment:
The algorithm can be used here, changing `y = f(dom)` with `y = enclose(f, dom, algorithm=algorithm)`.